tests/Config/ConfigurationLoaderTest.php

```diff
@@ -115,4 +144,31 @@ public function testReloadsOnExpiredCache(): void
         $this->assertNotNull($flagAgain);
         $this->assertEquals($flag, $flagAgain);
     }
+
+    public function testRunsWithoutBandits(): void
+    {
+        // Arrange: Load some flag data to be returned by the APIRequestWrapper
+        // Load mock response data
+        $flagsJson = json_decode(file_get_contents(self::MOCK_RESPONSE_FILENAME), true);
+
+        unset($flagsJson['bandits']); // Remove the Bandit Variations from the response
+        $flagResponse = json_encode($flagsJson);
+
+
+        $apiWrapper = $this->getMockBuilder(APIRequestWrapper::class)->disableOriginalConstructor()->getMock();
+        $apiWrapper->expects($this->exactly(1))
+            ->method('getUFC')
+            ->willReturn($flagResponse);
+        $apiWrapper->expects($this->exactly(1))
```

Comment:
we should expect this to be `0` as if the UFC indicates there are no bandits (by the `bandits` field not being present) we shouldn't even bother making the request. (Among other things, it will cut down on needless CDN costs)

Reply:
done